.github/workflows/e2e.yaml

```diff
@@ -82,6 +80,14 @@ jobs:
           working-directory: ./tasks/e2e
           spec: |
              cypress/integration/01-tutorial/*.spec.js
+             cypress/integration/03-storybook/*.spec.js
```

Comment:
🚀

The spec itself might need updating. It's been awhile.

Reply:
it works as expected; however we might want to figure out how to align it better w/ what happens in https://github.com/redwoodjs/redwood/blob/main/tasks/test-project/test-project

I reverted this change for now; as it does add some time to the e2e tests. Can easily bring it back and improve those specs if that is something we think is worth it.